soh/src/overlays/actors/ovl_En_Si/z_en_si.c

```diff
@@ -190,23 +199,28 @@ void EnSi_Draw(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-void Randomizer_GrantSkullReward(EnSi* this, GlobalContext* globalCtx) {
+void Randomizer_GetSkullReward(EnSi* this, GlobalContext* globalCtx) {
```

Comment:
names being hard strikes again

`GetSkullReward` implies to me this function should return the reward (GetItemEntry), but really it's setting it on the player

no great ideas for names come to mind, only thing i can think of at the moment is `PrepareSkullReward`

Reply:
update sounds good, either way works for me too